crates/rpc/src/handlers/committee.rs

```diff
@@ -0,0 +1,80 @@
+use std::vec;
+
+use ream_consensus::{constants::SLOTS_PER_EPOCH, misc::compute_start_slot_at_epoch};
+use ream_storage::db::ReamDB;
+use serde::Serialize;
+use warp::{
+    http::status::StatusCode,
+    reject::Rejection,
+    reply::{Reply, with_status},
+};
+
+use super::{BeaconResponse, state::get_state_from_id};
+use crate::types::{
+    errors::ApiError,
+    id::ID,
+    query::{EpochQuery, IndexQuery, SlotQuery},
+};
+
+#[derive(Debug, Serialize, Clone)]
+pub struct CommitteeData {
+    pub index: u64,
+    pub slot: u64,
+    pub validators: Vec<u64>,
+}
```

Comment:
can you fix the serialization, this is like the 4th time I commented on this, I shouldn't need to comment on the same problems every PR :pray: